### ISSUE TITLE
Drop down list

### ISSUE THREAD

**Issue by jlogan223** [custom-cards/button-card#867]
**Is your feature request related to a problem? Please describe.**
Would it be possible to add functionality to have a drop down list for input_selects?

**Describe the solution you'd like**
Custom: button-card isawesome,there's so much flexibility and that's why I use it for everything. The only thing I can't use it for is drop down list where an entity has more than 2 possible values, especially for stings.

**Describe alternatives you've considered**
I have looked at the Mushroom card and a HACS list card,but none have the customisation abilities that custom:button-cards have.

Something like this, where you can use text, numbers or even icons would be cool.

![dropdown-menu-design-1024x655 png](https://github.com/user-attachments/assets/6c8efba4-688c-45bc-9f49-f473884f073b)

**Comment by RomRider** [comment]
I don't think this belongs to button-card (as its name states, it's supposed to be a button 😃) but you could embed an input_select entity card inside button-card if needed and style it using card-mod.

**Comment by RomRider** [comment]
Duplicate of #224
